packages/graphql/src/schemaTypes/objectTypes/gql-Cohorts.ts

```diff
@@ -0,0 +1,10 @@
+import { inputObjectType, objectType } from 'nexus'
+
+export const Cohort = objectType({
+  name: 'Cohort',
+  description: 'used to distinguish one group of users from another',
+  definition (t) {
+    t.string('name', { description: 'name used to identify the cohort topic (e.g. "LoginBanner" ) ' })
```

Comment:
I already commented on this, but I keep getting tripped up thinking this is the name of an individual cohort rather than the name for a grouping of cohorts

Reply:
Any suggestions for a better name? My initial instinct was this would be something like `currentUserCohort`.

We also have. the field `cohort`, so when you query off this it is

```gql
cohort {
  cohort
  name
}
```

I wonder if the top level should be called something different. 

If we do want to include info about the current user cohort, I suppose it'd be on the `cachedUser` field

```gql
cachedUser {
  cohort {
    name
    # ...
  }
}
```

I don't think the naming is too bad, but I have not worked on ACI as much, so take my opinion with a grain of salt.

Reply:
I added documentation comments to useCohorts. Hopefully that helps with the names